.github/workflows/checks.yml

```diff
@@ -0,0 +1,70 @@
+on: [push, pull_request]
+
+name: Quick checks
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Run cargo check
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Run cargo test
+        uses: actions-rs/cargo@v1
+        continue-on-error: true  
+        with:
+          command: test
```

Comment:
I'm not familiar with Github actions (I used only  Travis on Gitlab CI), this sections executes the tests?

If yes, I guess they will fail because the [postgresql tests](https://github.com/galeone/bacup/blob/main/src/services/postgresql.rs#L172) require postgresql installed and running in the system, and I guess `ubuntu-latest` doesn't have it, am I right?

If yes, it is possible to add the installation step (or change the image maybe?)

Reply:
I would recommend separating postgres tests (and other services) to be run after docker containers are up. Would be a good practice not having test errors when some new person clones the repo and runs cargo test. 

There are options to run Github Actions (like [this](https://github.com/nektos/act)), but I think it's better to just mimic the CI test to run locally with a makefile (there you could also start the required containers for advanced tests).

Reply:
Is there a way to do this separation?

I like the idea of clinging the repo, execute `cargo test` and see all of them green - all the tests the require a third-party service (e.g. a database up and running) should only run if the appropriate containers are up.

Do you think it's feasible adding this kind of logic in the pull request?

Reply:
An easy way would be to ignore (using #[ignore]) those advanced tests and call them only after containers are ok. Could we agree on this issue treating just the basic CI flow, and create other issues for each service/container required?

Reply:
Totally agree (in the end, locally, I can just re-enable them until the docker env is ready)!